.travis/publish-gh-pages.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
This whole file can probably be removed if you don't intend to publish a `gh-pages` branch.